genesyscloud/resource_genesyscloud_outbound_ruleset_test.go

```diff
@@ -240,19 +240,20 @@ func TestAccResourceOutboundRuleset(t *testing.T) {
 				Config: generateOutboundContactList(
 					contactListResourceId1,
 					contactListName1,
-					"",
-					previewModeColumnName,
+					nullValue,
+					strconv.Quote(previewModeColumnName),
 					previewModeAcceptedValues,
 					columnNames,
 					automaticTimeZoneMapping,
-					"",
-					"",
+					nullValue,
+					nullValue,
 					generatePhoneColumnsBlock(
 						"Cell",
 						"cell",
-						"Cell",
+						strconv.Quote("Cell"),
```

Comment:
is there any test impact with this change from previous to now.

Reply:
No, I've been keeping an eye on the matrix tests. They are still passing. I'll make sure again before merging